CHANGES.md

```diff
@@ -1,5 +1,85 @@
 CHANGES.md lists intentional changes between the Strada (Typescript) and Corsa (Go) compilers.
 
+# Feature-Level Description
```

Comment:
If we are going to extend this file past JS changes, should we name the headers differently?